src/main/kotlin/service/user/Registration.kt

```diff
@@ -0,0 +1,48 @@
+package service.user
+
+import model.Users
+import org.jetbrains.exposed.sql.insert
+import org.jetbrains.exposed.sql.select
+import org.jetbrains.exposed.sql.transactions.transaction
+import org.mindrot.jbcrypt.BCrypt
+
+object Registration {
+    fun registerUser(registrationRequest: UserRegistrationRequest): UserAuthenticationResponse {
```

Comment:
Maybe split this up into several methods.

Reply:
Good point. Made it shorter :)